src/plugins/dashboard/public/dashboard_listing/hooks/use_dashboard_listing_table.tsx

```diff
@@ -110,6 +118,56 @@ export const useDashboardListingTable = ({
     goToDashboard();
   }, [dashboardSessionStorage, goToDashboard, useSessionStorageIntegration]);
 
+  const updateItemMeta = useCallback(
+    async (props: Pick<DashboardContainerInput, 'id' | 'title' | 'description' | 'tags'>) => {
+      await updateDashboardMeta(props);
+
+      setUnsavedDashboardIds(dashboardSessionStorage.getDashboardIdsWithUnsavedChanges());
+    },
+    [dashboardSessionStorage, updateDashboardMeta]
+  );
+
+  const contentEditorValidators: OpenContentEditorParams['customValidators'] = useMemo(
+    () => ({
+      title: [
+        {
+          type: 'warning',
+          fn: async (value: string, id: string) => {
+            if (id) {
+              try {
+                const [dashboard] = await findDashboards.findByIds([id]);
+                if (dashboard.status === 'error') {
+                  return;
+                }
+
+                const validTitle = await checkForDuplicateDashboardTitle({
+                  title: value,
+                  copyOnSave: false,
+                  lastSavedTitle: dashboard.attributes.title,
+                  isTitleDuplicateConfirmed: false,
+                });
+
+                if (!validTitle) {
+                  throw new Error(
+                    i18n.translate('dashboard.dashboardListingEditErrorTitle.duplicateWarning', {
```

Comment:
nit: Can we add this to the strings file `src/plugins/dashboard/public/dashboard_listing/_dashboard_listing_strings.ts` just to be consistent?